load_tests/_utils.py

```diff
@@ -0,0 +1,40 @@
+import random
```

Comment:
Looks like we will be duplicating a lot of the code here with standard tests. I think we should consider merging the two directories. But let us do that in a separate PR

Reply:
#1498